bundles/org.openhab.core.config.discovery.addon.ip/src/main/java/org/openhab/core/config/discovery/addon/ip/IpAddonFinder.java

```diff
@@ -614,18 +613,44 @@ private byte[] macBytesFrom(InetSocketAddress inetSocketAddress) throws SocketEx
 
     /**
      * Use the given format specifier to format an array of mac address bytes
-     * 
+     *
      * @param format a standard format specifier; optionally ends with a delimiter e.g. "%02x:" or "%02X"
      * @param bytes the mac address as an array of bytes
-     * 
+     *
```

Comment:
This empty line seems to cause a new SAT warning, 
```
checkstyle | 3 | 619 | style | JavadocMethodStyleCheck | There should be no empty lines between tags in a Javadoc comment of a method or constructor.
```

```suggestion
```

Reply:
Removed